win32_event_log/README.md

```diff
@@ -38,62 +51,86 @@ To see the most active channels, run the following command in PowerShell:
 
 <!-- xxx tabs xxx -->
+
 <!-- xxx tab "Events" xxx -->
 
 #### Event collection
 
-To collect Windows Event Logs as Datadog Events, configure channels under the `instances:` section of your `win32_event_log.d/conf.yaml` configuration file.
+To collect Windows Event Logs as Datadog events, configure channels under the `instances:` section of your `win32_event_log.d/conf.yaml` configuration file. 
 
-The agent can be configured to collect Windows Event Logs as Datadog Events in two ways. Each method has its own configuration syntax for channels and for filters (see [Filtering Events](?tab=events#filtering-events)). The legacy method uses WMI and is the default mode for an instance. The newer method uses the Event Log API. We recommend using the Event Log API because it has better performance. To use the Event Log API collection method, set `legacy_mode: false` in each instance. 
+The Datadog Agent can be configured to collect Windows Event Logs as Datadog events in two ways. Each method has its own configuration syntax for channels and for filters (see [Filtering Events](?tab=events#filtering-events)). 
 
-If `legacy_mode: false` is set, the `path` is required to be set in the `\win32_event_log.d\conf.yaml` file. If `legacy_mode` is not set or set to `true`, then at least one of the following filters must be set: `source_name`, `event_id`, `message_filters`, `log_file`, or `type`.
+* The latest method uses the Event Log API. Datadog recommends using the Event Log API because it has better performance than the legacy method below. 
 
-This example shows entries for the `Security` and `<CHANNEL_2>` channels:
+  </br> To use the Event Log API collection method, set `legacy_mode: false` in each instance. If `legacy_mode: false` is set, the `path` is required to be set in the `\win32_event_log.d\conf.yaml` file. 
 
-```yaml
-init_config:
-instances:
-  - # WMI - Legacy mode (default)
-    legacy_mode: true
-    log_file:
-      - Security
-
-  - # Event Log API (better performance)
-    path: Security
-    legacy_mode: false
-    filters: {}
-
-  - path: "<CHANNEL_2>" 
-    legacy_mode: false
-    filters: {}
-```
+  </br> This example shows entries for the `Security` and `<CHANNEL_2>` channels:
+
+  ```yaml
+  init_config:
+  instances:
+    - # Event Log API 
+      path: Security
+      legacy_mode: false
+      filters: {}
+
+    - path: "<CHANNEL_2>" 
+      legacy_mode: false
+      filters: {}
+  ```
+
+* The legacy method uses WMI and is the default mode for an instance. 
+  
+  </br> If `legacy_mode` is not set or set to `true`, then at least one of the following filters must be set: `source_name`, `event_id`, `message_filters`, `log_file`, or `type`.
+
+  </br> This example shows entries for the `Security` and `<CHANNEL_2>` channels:
+
+  ```yaml
+  init_config:
+  instances:
+    - # WMI (default)
+      legacy_mode: true
+      log_file:
+        - Security
+        
+    - path: "<CHANNEL_2>" 
+      legacy_mode: false
```

Comment:
this example mixes legacy/non-legacy mode.

Reply:
```suggestion
      legacy_mode: true
```